shopping-cart/shopping-cart-java/build.sbt

```diff
@@ -65,6 +74,7 @@ lazy val inventory = (project in file("inventory"))
 
 val lombok = "org.projectlombok" % "lombok" % "1.18.6"
 val postgresDriver = "org.postgresql" % "postgresql" % "42.2.5"
+val h2Driver = "com.h2database" % "h2" % "1.4.199" % Test
```

Comment:
Hmm, tests in the scala version of shopping-cart don't use `h2` right?

Reply:
no, that's a leftover. I have been trying many things